### DISCUSSION TITLE
Send CC events without MIDI Output Elements

### DISCUSSION THREAD

**Discussion by ojjo** [tttapa/Control-Surface#588] [Q&A] [answered]
Hello, 
I have a board with potentiometers, rotary encoders and buttons, already debounced and filtered with an arduino.

Data is transferred to a Teensy 3.6 with serial hardware to arrays of bytes (potValue[], buttonValue[], etc)

How to send bankable CC event like this example :

[https://tttapa.github.io/Control-Surface-doc/Doxygen/d9/d9d/MIDI-Controller-Finished-Example_8ino-example.html#a18](https://tttapa.github.io/Control-Surface-doc/Doxygen/d9/d9d/MIDI-Controller-Finished-Example_8ino-example.html#a18)

without instantiate a MIDI output element?

**Comment by tttapa** [comment, marked as answer]
Sending CC messages is easy: https://tttapa.github.io/Control-Surface-doc/Doxygen/d3/df7/midi-tutorial.html#midi_md-sending

Banks just add an offset to the address.

**Reply by ojjo** [reply]
Thank you!
